docs/howto/error-reporting.txt

```diff
@@ -282,7 +282,7 @@ following attributes and methods:
 
             import re
 
-            re.compile(r"API|TOKEN|KEY|SECRET|PASS|SIGNATURE|HTTP_COOKIE", flags=re.IGNORECASE)
+            re.compile(r"API|AUTH|TOKEN|KEY|SECRET|PASS|SIGNATURE|HTTP_COOKIE", flags=re.IGNORECASE)
```

Comment:
Should we add .. versionchanged?

Reply:
I pondered about this, I was undecided because our docs say that `versionchanged` is [for new features](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features), but given that you are also highlighting this, it may be worth adding it.

Will push this change shortly. Thank you!